shared/templates/create_services_enabled.py

```diff
@@ -20,23 +20,35 @@
 class ServiceEnabledGenerator(FilesGenerator):
 
     def generate(self, target, serviceinfo):
-        # get the items out of the list
-        servicename, packagename = serviceinfo
+        try:
+            # get the items out of the list
+            servicename, packagename, daemonname = serviceinfo
+        except ValueError as e:
+            print "\tEntry: %s\n" % serviceinfo
```

Comment:
same code as in [create_services_disabled.py](https://github.com/hex2a/scap-security-guide/blob/ebd2a1e8f85c34efee174b6dee46a78f03207403/shared/templates/create_services_disabled.py#L24), interpreter is explicitly set to python2